### DISCUSSION TITLE
Arc lines with OrthographicView

### DISCUSSION THREAD

**Discussion by Dmitriy1111111** [visgl/deck.gl#9212] [Q&A] [answered]
Hello to everyone! Is it expected that ArcLines layer is invisible on Orthographic view? Advise please! Thanks!

Example here - https://codepen.io/dgnedoy/pen/LYwbeKx

**Comment by Pessimistress** [comment, marked as answer]
The arc is clipped by the [near](https://deck.gl/docs/api-reference/core/orthographic-view#near) plane. If you set `getTilt:90` you can see it rendered.

**Reply by Dmitriy1111111** [reply]
@Pessimistress Thank you for clarification and fast response!
